app/main.py

```diff
@@ -1,8 +1,39 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int, average_rating: float, count_of_ratings: int):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+
+    def calculate_washing_price(self, car: Car) -> float:
+        return round((car.comfort_class * (self.clean_power - car.clean_mark) * self.average_rating)
+                     / self.distance_from_city_center, 1)
```

Comment:
There is a potential division by zero error in the `calculate_washing_price` method if `distance_from_city_center` is zero. Consider adding a check to prevent division by zero.